cli/worker.rs

```diff
@@ -610,7 +610,14 @@ impl CliMainWorkerFactory {
       options,
     );
 
-    if self.shared.subcommand.is_test_or_jupyter() {
+    if self.shared.subcommand.needs_test() {
+      worker.js_runtime.execute_script_static(
+        "40_testing.js",
```

Comment:
Rename this to `ext:cli/js/40_testing.js` and the error from the LSP should be resolved. Please do the same for `40_jupyter.js` as well